.circleci/config.yml

```diff
@@ -532,6 +543,16 @@ workflows:
               ignore: ['gh-pages']
           requires:
             - unit-tests-<< matrix.component_type >>
+      - publish-to-npm:
```

Comment:
Ensure packages are only published once all builds are successful.

Reply:
Out of interest, what happens if the build does fail? Does it just mean we'd have to run the publish manually or will it publish next time there's a green build?